diff --git a/node_modules/react-native-icons/android/build.gradle b/node_modules/react-native-icons/android/build.gradle
index 6dc3635..4d452cb 100755
--- a/node_modules/react-native-icons/android/build.gradle
+++ b/node_modules/react-native-icons/android/build.gradle
@@ -11,7 +11,7 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
+    compileSdkVersion 35
     buildToolsVersion "23.0.1"
 
     defaultConfig {
@@ -30,5 +30,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.16.+'
+    implementation "com.facebook.react:react-native:0.16.+"
 }
\ No newline at end of file
